.bazelrc

```diff
@@ -55,7 +55,7 @@ build:rocm --crosstool_top=@local_config_rocm//crosstool:toolchain
 build:rocm --define=using_rocm=true --define=using_rocm_hipcc=true
 build:rocm --define=xla_python_enable_gpu=true
 build:rocm --repo_env TF_NEED_ROCM=1
-build:rocm --action_env=TF_ROCM_AMDGPU_TARGETS="gfx803,gfx900,gfx906,gfx1010"
+build:rocm --action_env=TF_ROCM_AMDGPU_TARGETS="gfx900,gfx906,gfx908"
```

Comment:
@yashk2810 , I think it should be ```--action_env TF_ROCM_AMDGPU_TARGETS``` (drop =). right?

Reply:
Yeah, you are right. Can you change `TF_CUDA_COMPUTE_CAPABILITIES` too?

Reply:
Yes.